### DISCUSSION TITLE
Property creation - Etsy V3 - 513 Costom property - There was a problem with /attributes : Attributes not valid for property

### DISCUSSION THREAD

**Discussion by rubberfashion** [etsy/open-api#971] [Q&A]
Etsy V3
I want to create 2 propertys for a listing. 

Method: updateListingProperty
I want to create primary color:
PUT https://openapi.etsy.com/v3/application/shops/XXX/listings/XXX/properties/200
{'property_id': 200, 'property_name': 'Primary color', 'scale_id': 1, 'scale_name': '', 'value_ids': None, 'values': ['schwarz 0.4mm']}

Response: 
{'property_id': 200, 'property_name': 'Primary color', 'scale_id': 1, 'scale_name': 'US/CA', 'value_ids': [1138173629020], 'values': ['schwarz 0.4mm']}

Then i want to create next property:

I want to create the Size: (513 is costom property)
PUT https://openapi.etsy.com/v3/application/shops/XXX/listings/XXX/properties/513
{'property_id': 513, 'property_name': 'Size', 'scale_id': 1, 'scale_name': '', 'value_ids': None, 'values': ['S']}

Response: 
{'error': 'There was a problem with /attributes : Attributes not valid for property'}

What is the mistake???
BTW: Documentation is wrong.
updateListingProperty:
Request Body schema -> missing property_id, property_name
For new Property empty value_ids 
... 

Thanks a lot for help

**Comment by djego** [comment]
@rubberfashion  you could use id: 100 which is for property Size  maybe work it for you

**Comment by IanDanielM** [comment]
or you could rename to `sizeName` if using custom property
